scripts/pre_push_hook.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python2.7
```

Comment:
Could you add copyright etc. headers, same as for the other scripts in this folder?